src/main/scala/org/clulab/habitus/actions/HabitusActions.scala

```diff
@@ -186,20 +207,95 @@ class HabitusActions extends Actions {
   }
 
   val labelToAppropriateUnits = Map(
-    "Quantity" -> Set("t/ha", "kg/ha", "kg", "d", "cm"),
-    "AreaSize" -> Set("ha")
+    "Quantity" -> Set("t/ha", "kg/ha", "kg", "d", "cm", "mg/l", "kg n ha-1"),
+    "AreaSize" -> Set("ha"),
+    "YieldAmount" -> Set("t/ha", "kg/ha", "kg"),
+    "FertilizerQuantity" -> Set("kg/ha", "mg/l", "kg n ha-1")
   )
 
+  def hasLetters(string: String): Boolean = {
+    string.exists(ch => ch.isLetter || ch.toString == "/")
```

Comment:
Is that the same as ch == '/' ?

Reply:
Hi Keith, technically yes. I want to double check and address.

Reply:
At the moment changing `ch == '/'` does not work.

Reply:
yes, it does warn; `comparing values of types Char and String using `==` will always yield false` but having `ch.toString` works without warn.

Reply:
Do you notice that the quotation marks are now apostrophes?

Reply:
(It's different from Python.)

Reply:
yesss, sure. It is corrected. 

Thank you.